kubetest/kops.go

```diff
@@ -399,6 +399,12 @@ func (k kops) Up() error {
 	}
 	if k.adminAccess != "" {
 		createArgs = append(createArgs, "--admin-access", k.adminAccess)
+	} else {
+		var b bytes.Buffer
+		if err := httpRead("https://v4.ifconfig.co", &b); err != nil {
+			return err
```

Comment:
Perhaps log something about this?

Reply:
Er about using this value to set admin access

Reply:
done 👍